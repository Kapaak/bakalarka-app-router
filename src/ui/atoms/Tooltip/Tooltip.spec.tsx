import { render } from "@testing-library/react";

import { Tooltip } from "./Tooltip";

describe("Tooltip", () => {
  it("should render successfully", () => {
    const { baseElement } = render(
      <Tooltip content={null}>some tooltip</Tooltip>
    );

    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
