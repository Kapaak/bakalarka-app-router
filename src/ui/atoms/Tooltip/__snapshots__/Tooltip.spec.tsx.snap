// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip should render successfully 1`] = `
<body>
  <div>
    <button
      data-state="closed"
    >
      some tooltip
    </button>
  </div>
</body>
`;
